src/less/chart-legend/stories/chart-legend.stories.js

```diff
@@ -0,0 +1,31 @@
+export default { title: 'Chart-legend/Chart-legend' };
+
+export const basic = () => `
+<div legend-style="minimal" class="chart-legend">
+    <ul>
+        <li id="chart-legend-label-0" data-id="0" class="chart-legend__item--secondary">Value 1<span class="chart-legend--hidden">50.00%</span></li>
+        <li id="chart-legend-label-1" data-id="1" class="chart-legend__item--quinary">Value 2<span class="chart-legend--hidden">33.30%</span></li>
+        <li id="chart-legend-label-2" data-id="2" class="chart-legend__item--septenary">Value 3<span class="chart-legend--hidden">16.67%</span></li>
+    </ul>
+</div>
+`;
+
+export const table = () => `
+<div legend-style="table" class="chart-legend chart-legend__table">
+    <ul>
+        <li id="chart-legend-label-0" data-id="0" class="chart-legend__item--secondary">Value 1<span class="value">$ 3</span><span class="chart-legend--hidden">50.00%</span></li>
+        <li id="chart-legend-label-1" data-id="1" class="chart-legend__item--quinary">Value 2<span class="value">$ 2</span><span class="chart-legend--hidden">33.30%</span></li>
+        <li id="chart-legend-label-2" data-id="2" class="chart-legend__item--septenary">Value 3<span class="value">$ 1</span><span class="chart-legend--hidden">16.67%</span></li>
+    </ul>
+</div>
+`;
+
+export const patternfill = () => `
+<div legend-style="minimal" patternfill="" class="chart-legend">
```

Comment:
Should these be data attributes? e.g. `data-chart-legend-style` and `data-chart-patternfill`?

Reply:
I think I need to add these to the ignore attributes so they don't get added, I had copy pasted from a live running version from ebayui-core and glossed over these being added.